generators/app/templates/src/hooks/useRequest.test.tsx

```diff
@@ -31,6 +30,10 @@ const MockService = {
   fetchFoo: () => Promise.resolve(successResponse)
 };
 
+const LoadingMockService = {
+  fetchFoo: () => new Promise<typeof successResponse>(() => {})
```

Comment:
`fetchFoo: jest.fn(() => new Promise<typeof successResponse>(jest.fn());` does this works?